### DISCUSSION TITLE
Renault ZOE solar charging "issue"

### DISCUSSION THREAD

**Discussion by nathanielakkermans** [serkri/SmartEVSE-3#252] [General]
I have been using this firmware for a while now with the solar function and I must say that it works very well!
I have a Renault Zoë and if you plug it in in the evening and the EVSE is in solar mode. then the car will of course not start charging, but the car would like to start charging as soon as the plug is plugged into the car. but because it does not get a "go" from the EVSE (because there is no sun) then the car will go into an error state over time and when the sun rises in the morning it will not start charging. but if I let the car charge in normal mode for a couple of minutes and then switch it to solar, the car will accept it and you can leave it plugged in for days and the car will charge properly when the EVSE notices that there is solar power left.
This is not really a problem with the EVSE of course, but with the charging implementation in the car, but perhaps it would be an idea/option to come up with something as soon as you plug in a car in solar mode that it will first charge for a few minutes regardless. whether there is solar energy left or not. and then wait to see if there is any power left or not. I don't know if more car brands have this "problem", but perhaps it could be a nice addition to the functionality of the project.
Until then I just plug it in in normal or smart mode and then switch it to solar, not a lot of work either.

**Comment by dingo35** [comment]
We have had reports that v3.5.0-RC1 behaves much better with the Zoë, so you could try that. The French lady has her antics...

**Comment by nathanielakkermans** [comment]
thanks for the quick response, I will try this firmware version!
